doc/index.rst

```diff
@@ -3,39 +3,67 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+
+========================
+QGreenland Documentation
+========================
```

Comment:
This was oddly critical. We struggled for at least an hour with "Why is the first chapter's header getting swallowed, so its children look like chapters?"